### PR TITLE
Profile screen visual update

### PR DESCRIPTION
**Pull request by UnavailableCode**
image
New : 

- This PR updated the visual style of the profile according to the Figma.

Bugfixes : 

- Fixed an issue with the creation of the UserProfileViewModel that made the creation of the profileOverview composable impossible and that induced a lot of code copy for the preview. This will still need to be fixed when the profile will be linked to the viewModel

### CODE REVIEW

File: app/src/main/java/com/example/triptracker/view/profile/UserProfileOverview.kt [hunk 1]
```diff
@@ -22,33 +22,60 @@ import androidx.compose.material3.Icon
...
+ */
+@Composable
+fun UserProfileOverview(
+    // userProfileViewModel: UserProfileViewModel = UserProfileViewModel(),
+    navigation: Navigation
+) {
+  // UID of the user that needs to be fetched from ??, used to fetch the profile from the database
+  // val uid = "TODO"
+  // val profile = userProfileViewModel.userProfileList.getUserProfile(uid)
+  // val url = profile.profileImageUrl
```
Comment (SpaceMercury) [inline app/src/main/java/com/example/triptracker/view/profile/UserProfileOverview.kt:74]: Are these a work in progress/ need to be implemented? If so we need to add to the Backlog.

**Review by SpaceMercury — APPROVED** [review]
Everything looks good, UI is clean and looks like the Figma we designed. Just make sure to maybe clarify the comments (on the pr description), but maybe I am nitpicking a bit. Otherwise, looks good, great job!

File: app/src/main/java/com/example/triptracker/view/profile/UserProfileOverview.kt (hunk 1 above)
Comment (UnavailableCode) [reply app/src/main/java/com/example/triptracker/view/profile/UserProfileOverview.kt:77]: These are code that will be used later when the Profil will be linked to the UserProfile viewModel

File: app/src/main/java/com/example/triptracker/view/profile/UserProfileOverview.kt [hunk 2]
```diff
@@ -22,33 +22,60 @@ import androidx.compose.material3.Icon
...
+/**
+ * Composable displaying the profile overview featuring information about follower, following. 4
+ * clickable buttons are available to access to the MyTrips, Favorites, Friends and settings views.
+ *
+ * @param userProfileViewModel: The view model of the UserProfile
+ * @param navigation: The navigation of the app to switch between different views
+ */
+@Composable
+fun UserProfileOverview(
+    // userProfileViewModel: UserProfileViewModel = UserProfileViewModel(),
```
[the remark this answers is not in the document]
Comment (UnavailableCode) [reply app/src/main/java/com/example/triptracker/view/profile/UserProfileOverview.kt:71]: Exactly !

**Pull request merged**